backend/server/server.go

```diff
@@ -38,6 +40,7 @@ func Start() {
 	db.Init()
 	migration.Init()
 	app.Init()
+	lang.Init()
 	validator.Init()
 	gob.Register(psession.SessionUser{})
 	cache.Init()
```

Comment:
The provided code changes seem to be adding an additional initialization step for setting up language translations using the `lang` package from `github.com/1Panel-dev/1Panel/backend/init`. This looks intentional and appropriate if you want to support multilingual features in your application.

Here are some considerations:

1. **Resource Management**: Ensure that the initialized languages are properly managed during runtime to avoid memory leaks or conflicts with other language settings.

2. **Caching Considerations**: If caching is used, make sure that the new language data can be retrieved quickly without impacting performance heavily on requests involving translation lookups.

3. **Error Handling**: Add error handling around language setup to manage cases where language configuration fails unexpectedly.

4. **Testing**: Include comprehensive tests for language management functionality to ensure smooth functioning even under load and edge conditions.

Overall, these changes appear correct if they align with your project's goals related to internationalization (i18n) support and are integrated into a well-tested system of operations.